src/components/autosuggest/autosuggest.spec.js

```diff
@@ -75,419 +25,428 @@ describe('script: autosuggest', () => {
+                );
+                expect(instructionsDisplayStyle).toBe('block');
+                const listboxDisplayStyle = await page.$eval('.ons-js-autosuggest-listbox', (node) => getComputedStyle(node).display);
+                expect(listboxDisplayStyle).toBe('block');
+                const statusDisplayStyle = await page.$eval('.ons-js-autosuggest-aria-status', (node) => getComputedStyle(node).display);
+                expect(statusDisplayStyle).toBe('block');
+            });
         });
-    });
 
-    describe('when the user presses the "Down" arrow key', () => {
-        it('navigates to the first suggestion initially', async () => {
-            await setTestPage('/test', renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST));
+        describe('WHEN: the user presses the "Down" arrow key', () => {
+            test('THEN: it navigates to the first suggestion initially', async () => {
+                await setTestPage('/test', renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST));
 
-            await page.type('.ons-js-autosuggest-input', 'Unite', { delay: 20 });
-            await page.keyboard.press('ArrowDown');
+                await page.type('.ons-js-autosuggest-input', 'Unite', { delay: 20 });
+                await page.keyboard.press('ArrowDown');
 
-            const selectedOption = await page.$eval('.ons-autosuggest__option--focused', (node) => node.textContent);
-            expect(selectedOption.trim()).toBe('United States of America');
-        });
+                const selectedOption = await page.$eval('.ons-autosuggest__option--focused', (node) => node.textContent);
+                expect(selectedOption.trim()).toBe('United States of America');
+            });
 
-        it('navigates to the suggestion below the current suggestion', async () => {
-            await setTestPage('/test', renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST));
+            test('THEN: it navigates to the suggestion below the current suggestion', async () => {
+                await setTestPage('/test', renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST));
 
-            await page.type('.ons-js-autosuggest-input', 'Unite', { delay: 20 });
-            await page.keyboard.press('ArrowDown');
-            await page.keyboard.press('ArrowDown');
+                await page.type('.ons-js-autosuggest-input', 'Unite', { delay: 20 });
+                await page.keyboard.press('ArrowDown');
+                await page.keyboard.press('ArrowDown');
 
-            const selectedOption = await page.$eval('.ons-autosuggest__option--focused', (node) => node.textContent);
-            expect(selectedOption.trim()).toBe('United States Virgin Islands');
-        });
+                const selectedOption = await page.$eval('.ons-autosuggest__option--focused', (node) => node.textContent);
+                expect(selectedOption.trim()).toBe('United States Virgin Islands');
+            });
```

Comment:
We could reduce the repetition here by moving some of this into the WHEN which would also make more sense with the WHEN description